### DISCUSSION TITLE
Significant buffering with various channels under the China country playlist

### DISCUSSION THREAD

**Discussion by hwhlee** [iptv-org/community#69] [Show and tell]
### Broken Link

https://iptv-org.github.io/iptv/countries/cn.m3u

### What happened to the stream?

Constantly interrupts/lagging

### Notes

Playlist being used: https://iptv-org.github.io/iptv/countries/cn.m3u
Client: Kodi (PVR IPTV Simple Client)
My Location: Sydney, Australia
Internet connection: 100mbps down / 20mpbs up

Examples of some channels with significant buffering (to the point of being unwatchable):
1) CCTV-1综合 | http://39.134.115.163:8080/PLTV/88888910/224/3221225618/index.m3u8
2) CCTV-2财经 | http://39.134.115.163:8080/PLTV/88888910/224/3221225619/index.m3u8
3) CCTV-5体育 | http://39.134.115.163:8080/PLTV/88888910/224/3221225633/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

**Comment by BellezaEmporium** [comment]
These channels I believe are self-hosted using a China Telecom package / subscription, and it seems they're redirecting the flux from their provider to this kind of links, which might explain the constant buffering.

I, having 800 mbps down / 600 mbps up, also have difficulties watching them.

**Reply by hwhlee** [reply]
I see.  Would a VPN service that enables accessing local Chinese content get around this problem?
